src/TagsInput/components/DeleteButton.js

```diff
@@ -0,0 +1,37 @@
+// @flow
+
+import * as React from 'react';
+
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+import { Touchable } from '../../Touchable';
+import type { StylePropType } from '../../PlatformStyleSheet/StyleTypes';
+
+import { Icon } from '../../Icon';
+
+type Props = {|
+  +onPress: () => void,
+  +style: StylePropType,
+  +disabled?: boolean,
+|};
+
+class DeleteButton extends React.PureComponent<Props> {
+  render() {
+    const { onPress, disabled, style } = this.props;
+    return (
+      <Touchable
+        testID="clear-button"
```

Comment:
Just to be consistent with the naming of your component, could you change it to "delete-button"?